### ISSUE TITLE
NoMethodError in Management API

### ISSUE THREAD

**Issue by kriskelly** [openware/peatio#1010]
I'm trying to use the Management API and for some reason I've got this error message when it tries to verify the JWT:

```
Started POST "/management_api/v1/deposits" for 127.0.0.1 at 2018-04-30 12:58:13 -0700
#<NoMethodError: undefined method `fetch' for nil:NilClass>
/Users/kriskelly/myproject/workbench/app/peatio/app/api/management_api_v1/jwt_authentication_middleware.rb:60:in `check_jwt!'
/Users/kriskelly/myproject/workbench/app/peatio/app/api/management_api_v1/jwt_authentication_middleware.rb:13:in `before'
/Users/kriskelly/.rvm/gems/ruby-2.5.0/gems/grape-1.0.1/lib/grape/middleware/base.rb:29:in `call!'
/Users/kriskelly/.rvm/gems/ruby-2.5.0/gems/grape-1.0.1/lib/grape/middleware/base.rb:24:in `call'
/Users/kriskelly/.rvm/gems/ruby-2.5.0/gems/grape-1.0.1/lib/grape/middleware/error.rb:37:in `block in call!'
/Users/kriskelly/.rvm/gems/ruby-2.5.0/gems/grape-1.0.1/lib/grape/middleware/error.rb:36:in `catch'
/Users/kriskelly/.rvm/gems/ruby-2.5.0/gems/grape-1.0.1/lib/grape/middleware/error.rb:36:in `call!'
/Users/kriskelly/.rvm/gems/ruby-2.5.0/gems/grape-1.0.1/lib/grape/middleware/base.rb:24:in `call'
```
Apparently the `security_configuration` value is nil, which causes it to fail.

**Comment by yivo** [comment]
Hi @kriskelly 

You need to carefully read config/management_api_v1.yml and properly configure https://github.com/rubykube/peatio/blob/master/config/templates/management_api_v1.yml.erb#L64

**Comment by kriskelly** [comment]
@yivo I have had that yml file correctly configured to my knowledge. Here's a copy of what it looks like:

```
keychain:
  backend.mydomain.com:
    algorithm: RS256
    value: LS0tL.... (base64-encoded public key left out here)

jwt: {}

scopes:
  read_deposits:
    permitted_signers: ['backend.mydomain.com']
    mandatory_signers: ['backend.mydomain.com']
  write_deposits:
    permitted_signers: ['backend.mydomain.com']
    mandatory_signers: ['backend.mydomain.com']
  read_withdrawals:
    permitted_signers: ['backend.mydomain.com']
    mandatory_signers: ['backend.mydomain.com']
  write_withdrawals:
    permitted_signers: ['backend.mydomain.com']
    mandatory_signers: ['backend.mydomain.com']
```

Is there something wrong with that config? I can verify that the `security_configuration` is being set properly in the initializer, it just doesn't seem to be set when I run the app. I'm running the app locally via puma with `RAILS_ENV=development`, do you think there could be a problem there?

**Comment by yivo** [comment]
Resolved via #1014.
